components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint/src/main/webapp/challenge-question-view.jsp

```diff
@@ -105,7 +105,7 @@
                             <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 form-group">
                                 <input id="securityQuestionAnswer" name="securityQuestionAnswer" type="password"
                                        class="form-control"
-                                       tabindex="0" autocomplete="off" required/>
```

Comment:
Why do we need to change like this?

Reply:
autocomplete="off" doesn't solve the issue of the input box getting auto filled, but autocomplete="new-password" works

Reply:
`autocomplete="off"` used to work earlier.. Why do we need to change this now?

Reply:
I tried out the scenario with autocomplete="off" but did not work, so I found this solution https://stackoverflow.com/questions/12374442/chrome-ignores-autocomplete-off and tried out the suggestion and worked for me

Reply:
At glance, that thread contains multiple answers with various solutions.

Can you point me to a verified source that confirms above solution works on all browsers and all versions?

Reply:
the MDN web docs regarding the matter: https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#The_autocomplete_attribute_and_login_fields

chromium projects: https://www.chromium.org/developers/design-documents/form-styles-that-chromium-understands